committee.rst

```diff
@@ -0,0 +1,152 @@
+  our repository up-to-date and running our selection process.
+
+* The committee has one nudger. The nudger's job is to make sure other
+  committee members are fulfilling their duties by nudging them (often
+  in public) to do so.
+
+The co-chairs and secretary
+positions remain with an individual until they decide to relinquish the
+role, though a majority vote among the committee can strip a member of
+their co-chair/secretary position.
+
+The nudger position rotates among members every six months.
+In the event that the nudger ends up needing nudging (by any other member
+of the committee), the nudger should step down and volunteer to do a better
+job in some future six months; in the event that they still struggle to
+stay on top of their nudging duties, it may be a sign that they are too
+busy for committee work and should step down.
+
+The secretary is in charge of managing who the nudger is, by creating
+a rotation for this position.
+
+Guidelines for Communication
+----------------------------
+
+All members of the Steering Committee agree to abide by the
+`Guidelines for Respectful Communication <https://github.com/ghc-proposals/ghc-proposals/blob/master/GRC.rst>`_. As the guidelines document
+describes, if one of us errs in our communication, please
+opt to bring up the matter with that individual directly.
+If that is not feasible, then please bring it up with the
+(co-)chair of the committee. Everyone should feel welcomed
+to contribute to the growth of GHC and Haskell, and we eagerly
+want to know when our actions cause someone not to feel that
+way.
+
+Method of Communication
+-----------------------
+
+Most official committee business takes place via the
+`ghc-steering-commitee <https://mail.haskell.org/mailman/listinfo/ghc-steering-committee>`_ mailing list. The archives of this list are public, and
+any member of the community may subscribe. It is intended that the list
+be used by the committee, though community members might be asked
+to post there during relevant conversations.
+
+Any business not on the public mailing list will be of a sensitive nature,
+most likely pertaining to individuals. In particular, discussions of selecting
+new members and of violations of the guidelines for respectful communication
+will not appear on the mailing list.
+
+Term Limits and Committee Selection
+-----------------------------------
+
+Any community thrives best by continuing to refresh itself with new members.
+Accordingly, membership on the committee comes with a three-year term. Terms
+begin/end in October. Toward the end of an individual's term, the secretary of
+the committee will put out a public call for nominations for people to join
+the committee. Self-nominations are welcome and expected. The member whose
+term is ending is free to re-nominate themselves. Nominations include a brief
+bio of the nominee and reasons why they would be appropriate for the
+committee.
+
```

Comment:
The main criteria for becoming a member of the committee are:
* A track record of constructive contribution to public discussion of GHC proposals
* A track record of constructive contribution to one or more of the communities outlined above (users, tool authors etc)
* An expressed willingness to respond to GHC proposals in a timely manner.

These criteria are not exhaustive -- nominations and self-nominations are free to strengthen the case in whatever way they deem appropriate --- but a record of thoughtful contribution carries most weight.
*